vmware_exporter/vmware_exporter.py

```diff
@@ -636,10 +636,12 @@ def _vmware_get_vm_perf_manager_metrics(self, vm_metrics):
         content = yield self.content
 
         if len(specs) > 0:
-            results, labels = yield parallelize(
-                threads.deferToThread(content.perfManager.QueryStats, querySpec=specs),
-                self.vm_labels,
-            )
+            chunks = [specs[x:x+5000] for x in range(0, len(specs), 5000)]
```

Comment:
interesting :+1: but, how about making this value configurable with a default (5000?), this can help others to tune based on their specific needs/system capabilities.

Reply:
We can define a default value to 5000 and can be overloaded if define in config.yml. Do you think it's a good idea ?

Reply:
yep, that would be my approach for this.

@Jc2k @pryorda thoughts?

Reply:
@dannyk81 I completely agree. A configuration would be nice here. @alecorps have you watched the difference in the completion times?

Reply:
It never works if i don't split the specs list. I tried with 7000, 10000, it works but i take more time. Shorter is the specs list, shorter is the duration.